nixio/block.py

```diff
@@ -83,11 +83,19 @@ def create_multi_tag(self, name="", type_="", positions=0,
             return self.multi_tags[id]
 
         util.check_entity_name_and_type(name, type_)
-        util.check_entity_input(positions)
         multi_tags = self._h5group.open_group("multi_tags")
         if name in multi_tags:
             raise exceptions.DuplicateName("create_multi_tag")
-        mtag = MultiTag._create_new(self, multi_tags, name, type_, positions)
+        if not isinstance(positions, DataArray):
+            da_name = "{}-positions".format(name)
+            positions = self.create_data_array(da_name, "{}-positions".format(type_),
+                                        data=positions)
+        if not isinstance(extents, DataArray) and extents is not None:
+            da_name = "{}-extents".format(name)
+            extents = self.create_data_array(da_name,
+                                        "{}-extents".format(type_)
+                                            , data=extents)
```

Comment:
```suggestion
            extents = self.create_data_array(da_name,
                                             "{}-extents".format(type_),
                                             data=extents)
```

See also above comment about the DataArray creation exceptions.